From 6ef29fd2e2284d7546c63718c6396fd1f3ca28e6 Mon Sep 17 00:00:00 2001
From: George T Kramer <george.t.kramer@intel.com>
Date: Fri, 9 Aug 2019 16:04:53 -0700
Subject: [PATCH 2/8] Do not build kojivm

This is not a required service and to use it, dependencies on libvirt
are pulled in, making this a larger package than desired.
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 995b23f..720fd46 100644
--- a/Makefile
+++ b/Makefile
@@ -1,6 +1,6 @@
 NAME=koji
 SPECFILE = $(firstword $(wildcard *.spec))
-SUBDIRS = hub builder koji cli util www plugins vm
+SUBDIRS = hub builder koji cli util www plugins
 
 ifndef PYTHON
 export PYTHON=python2
-- 
2.25.1

